db/tests/dependents/test_dependents.py

```diff
@@ -161,3 +166,29 @@ def test_dependents_graph_max_level(engine_with_schema):
     dependents_by_level = sorted(t0_dependents_graph, key=lambda x: x['level'])
     assert dependents_by_level[0]['level'] == 1
     assert dependents_by_level[-1]['level'] == 10
+
+
+def test_views_as_dependents(engine_with_schema, library_db_tables, library_tables_oids):
+    engine, schema = engine_with_schema
+    metadata = MetaData(schema=schema, bind=engine)
+
+    publications = library_db_tables['Publications']
+    new_publications_view = select(publications).where(publications.c['Publication Year'] >= 2000)
+    create_view('NewPublications', new_publications_view, metadata)
+    metadata.create_all(engine)
+
+    publications_oid = library_tables_oids['Publications']
+    publications_dependents_graph = get_dependents_graph(publications_oid, engine)
+    publications_view_dependents = _get_object_dependents_by_type(publications_dependents_graph, publications_oid, 'view')
+
+    assert len(publications_view_dependents) == 1
```

Comment:
Please make sure you are also testing if the correct view is being returned by checking if the property of the view such as the name is as expected.